src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityLargeBoiler.java

```diff
@@ -409,10 +159,10 @@ protected BlockPattern createStructurePattern() {
                 .where('P', states(boilerType.pipeState))
                 .where('X', states(boilerType.fireboxState).setMinGlobalLimited(4)
                         .or(abilities(MultiblockAbility.IMPORT_FLUIDS).setMinGlobalLimited(1))
-                        .or(abilities(MultiblockAbility.IMPORT_ITEMS).setMinGlobalLimited(1)))
+                        .or(abilities(MultiblockAbility.IMPORT_ITEMS).setMinGlobalLimited(1))
+                        .or(autoAbilities())) // muffler, maintenance
```

Comment:
I also think that looks weird. But it also makes sort of sense, it's the fuel burning that should emit particles, which is done on the bottom buses/hatches.
But maybe it's better to say screw it and go with what would look better: muffler hatch at the very top. I don't know the answer.

Reply:
The fireboxes are what is actually burning the input fuel. It makes sense for the smoke to come out of them. 

A muffler in the main part of the boiler means that the boiler is filling with smoke (instead of steam) and expelling it. Alternatively, it could mean the boiler is expelling (black) steam, which is also weird given the goal is to produce it in a usable manner. Both of these cases don't make much sense.